<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileClass</class>
 <widget class="QWidget" name="fileClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fileClass</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Входной файл:</string>
   </property>
   <widget class="QLineEdit" name="lineFilePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bOpen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkCycle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Циклическая передача</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать поток</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkRadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Радиосигнал</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
